azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/di/DependencyInjectionContainerImpl.kt

```diff
@@ -214,10 +214,11 @@ internal class DependencyInjectionContainerImpl(
     // Initial State
     private val initialState by lazy {
         AppReduxState(
-            configuration.callConfig?.displayName,
-            localOptions?.isCameraOn == true,
-            localOptions?.isMicrophoneOn == true,
-            localOptions?.audioVideoMode ?: CallCompositeAudioVideoMode.AUDIO_AND_VIDEO
+            displayName = configuration.callConfig?.displayName,
+            cameraOnByDefault = localOptions?.isCameraOn == true,
+            microphoneOnByDefault = localOptions?.isMicrophoneOn == true,
+            avMode = localOptions?.audioVideoMode ?: CallCompositeAudioVideoMode.AUDIO_AND_VIDEO,
+            skipSetupScreen = localOptions?.isSkipSetupScreen ?: false,
```

Comment:
Is isSkipSetupScreen a nullable property? Can we check like we did for isCameraOn to keep consistency?

Reply:
localOptions can be null so `localOptions?` may result in whole `localOptions?.isSkipSetupScreen` be null

Reply:
Sounds good :)

Reply:
Discussed with Mohtasim and we decided to `?:` to `isCameraOn` and `isMicrophoneOn` as well